### PR TITLE
Added support for Ubuntu

### PR DESCRIPTION
**Pull request by 3isenHeiM**
Automatic validation of the GPG key with the ubuntu keyserver.

### CODE REVIEW

File: distrib-dl [hunk 1]
```diff
@@ -329,6 +332,35 @@ function download_fedora() {
     fi
 }
 
+function download_ubuntu() {
+    # Official netinstall image without firmwares (disabled)
+    #debian_base_url="https://cdimage.debian.org/debian-cd/current/${architecture}/iso-cd/"
+    #debian_iso_filename="debian-${debian_version}-${architecture}-netinst.iso"
+    # Unofficial image including firmwares
```
Comment (nodiscc) [inline distrib-dl]: please remove the 4 commented out lines above, there are no alternative images for ubuntu

**Review by nodiscc — CHANGES_REQUESTED** [review]
`ubuntu` should be added to the `Available distributions:` list L58

Thanks!

**Comment by 3isenHeiM** [comment]
Changes implemented in a new commit.

**Review by nodiscc — APPROVED** [review]
thank you

**Pull request merged**